modules/ossm-tutorial-bookinfo-verify-install.adoc

```diff
@@ -25,11 +25,26 @@ Before configuring your application, verify that it successfully deployed.
 ----
 $ oc get pods -n bookinfo
 ----
-
-. Run the following command to retrieve the URL for the product page:
++
+You should receive a response `200`.
++
+** Alternatively, you can open `http://$GATEWAY_URL/productpage` in your browser.
+** You can also verify that all pods are ready with this command:
 +
 [source,terminal]
 ----
 echo "http://$GATEWAY_URL/productpage"
 ----
-. Paste the output in a web browser to verify the Bookinfo product page is deployed correctly.
++
+You should see output similar to the following:
```

Comment:
Just to double-check the formatting: does this output belong to the last command only? And not to the whole Step 2?

Reply:
For the first command (oc get pods) you should receive a response "200".  The formatted output at the end should for the echo command.

But it's been so long since I've touched this particular topic, that now I'm going to spin up a cluster just to double check.    =P

Reply:
Thanks for this comment, I went back and ran through the procedure and somehow a couple of lines had gotten moved around and were incorrect.  It's fixed now.

Reply:
Glad to help (even if accidentally) :) Cool that it's fixed.